api/client/search.go

```diff
@@ -60,7 +71,14 @@ func (cli *DockerCli) CmdSearch(args ...string) error {
 	w := tabwriter.NewWriter(cli.out, 10, 1, 3, ' ', 0)
 	fmt.Fprintf(w, "NAME\tDESCRIPTION\tSTARS\tOFFICIAL\tAUTOMATED\n")
 	for _, res := range results {
-		if (*automated && !res.IsAutomated) || (int(*stars) > res.StarCount) {
+		if filterOptions["is-automated"] == "true" && !res.IsAutomated {
```

Comment:
@thaJeztah did and commited. I don't actually understand why tests don't pass. Should a new CLI flag be merged into Docker before writing tests for it?